lit/tests/OMTest.sc

```diff
@@ -0,0 +1,65 @@
+// RUN: not scala-cli --server=false --java-home=%JAVAHOME --extra-jars=%RUNCLASSPATH --scala-version=%SCALAVERSION %s -- panama 2>&1 | FileCheck %s -check-prefix=NO-COMPILER-PLUGIN
```

Comment:
compiler-plugin is required.

Reply:
This line is copied from `SmokeTest.scala` for testing the expected error if compiler-plugin is not loaded.